auto_submit/lib/model/pr_type.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+enum PullRequestType {
+  merge(name: 'merge'),
+  rebase(name: 'rebase'),
+  revert(name: 'revert');
+
+  const PullRequestType({required this.name});
+
+  final String name;
```

Comment:
Is the name needed? It seems like we could do something like:

```
enum PullRequestType {
  merge,
  rebase,
  revert,
}

Reply:
We could if the enum supports some sort of stringification of the values.

Reply:
```dart
print(PullRequestType.merge.name); // prints 'merge'
```

Reply:
Yeah you beat me to it! LOL! I already have that change just had not pushed it.